aeneas/src/vst/Parser.v3

```diff
@@ -185,18 +185,28 @@ component Parser {
 		var id = parseIdent(p, parseTypeParam);
 		var params = parseOptionalParams(p, parseVariantCaseParam);
 		var desugar = VariantDesugaring.new(isPrivate, id, params);
-		var repHint = parseRepHint(p);
+		var repHints = parseRepHints(p);
 		var decl = desugar.synthesizeTopLevelClass(parseVariantCases(p, desugar));
-		decl.repHint = repHint;
+		decl.repHints = repHints;
 		file.classes.put(decl);
 		// lift case classes to top-level of file
 		for (cdecl in decl.cases) {
 			if (cdecl.decl != decl) file.classes.put(cdecl.decl);
 		}
 		return true;
 	}
-	def parseRepHint(p: ParserState) -> VstRepHint {
-		var list: List<VstIdent<void>>;
+	def parsePacking(p: ParserState) -> string {
```

Comment:
This eats far too much as a token, e.g. all the normal lexical delimiters like `(){}`, and keywords and so on. We need to be conservative by default and only parse packings that we intend to support. E.g. I think we want to support packing expressions that reference a declared packing (with arguments as variables or constants), but not much else.